ext/sodium/config.m4

```diff
@@ -1,5 +1,5 @@
 dnl $Id$
-dnl config.m4 for extension libsodium
+dnl config.m4 for extension sodium
 
 PHP_ARG_WITH(libsodium, for libsodium support,
 [  --with-libsodium[[=DIR]]  Include libsodium support])
```

Comment:
The extension is "sodium", and thus we should use some --with-sodium to be consistent, (enable/with is not revelant here)

See ext/zip for ex, it have "--enable-zip" for the extension and "--with-libzip" for the library.